src/App.js

```diff
@@ -1,25 +1,62 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    name: '0',
```

Comment:
```suggestion
    name: '0',
```
It's number or string? If you are using numbers (with Math.random), so no need to convert them into strings